README.md

```diff
@@ -1,123 +1,96 @@
-[hub.docker.io](https://registry.hub.docker.com/u/jujusolutions/jujubox/):
-
-### Linux
-You can clone the repository from github.
+If you want to see the code you can download the jujubox project from 
+[github](https://github.com/juju-solutions/jujubox) and build it from source.
 
 ```
 git clone https://github.com/jujusolutions/jujubox.git && cd jujubox
-
+docker build -t jujusolutions/jujubox ./
 ```
 
-Or download and install the container directly from hub.docker.io:
+Or simply pull the jujubox container from
+[hub.docker.io](https://registry.hub.docker.com/u/jujusolutions/jujubox/).
 
 ```
-sudo docker run  -ti jujusolutions/jujubox
+docker pull jujusolutions/jujubox
 ```
 
+# Run jujubox
 
-# To Run
-
-There are two ways to run the docker container, from the github
-repository (build from source) or from the container directly from the
-hub.docker.io.
+There are several different ways to run jujubox depending on what you are 
+trying to accomplish. 
 
+## Running jujubox to evaluate Juju
 
-### Linux
+If you do not already have Juju installed and want to check this container out 
+you can simply run the container in an interactive (-i) tty (-t) session to use
+the Juju tools:
 
-#### From source:
 ```
-sudo docker build -t jujubox ./
-sudo docker run -i -t jujubox:latest
+docker run --rm -it jujusolutions/jujubox 
 ```
 
-#### Use hub.docker.io:
-```
-sudo docker run  -ti jujusolutions/jujubox
-```
-
-If you already have juju installed, run the container using your
-existing ${JUJU_HOME} directory (~/.juju):
+## Running jujubox if you have Juju installed on host
 
-```
-sudo docker run --rm -v ${JUJU_HOME}:/home/ubuntu/.juju -it jujusolutions/jujubox
-```
+If you already have Juju installed on the host machine, you can mount the 
+existing host directories to the container using volume mounts (-v). Run the 
+container using your volume mounts to your existing ${JUJU_HOME} directory 
+(~/.juju) and your existing ${JUJU_REPOSITORY} directory (where the local 
+charms are held):
 
-### OS X
-Remember that boot2docker runs as root, so don't use `sudo` for the docker
-commands:
 ```
-docker run -ti jujusolutions/jujubox
+docker run --rm -it \
+  -v ${JUJU_HOME}:/home/ubuntu/.juju \
+  -v ${JUJU_REPOSITORY}/builds:/home/ubuntu/builds \
+  -v ${JUJU_REPOSITORY}/trusty:/home/ubuntu/trusty \
+  -v ${JUJU_REPOSITORY}/xenial:/home/ubuntu/xenial \
+  jujusolutions/jujubox
 ```
 
-# Once it's up
-
-All your usual juju commands should work. You can use Quickstart if
-you need to set up your cloud credentials:
-
-    juju quickstart -i
-
-
-# Other goodies
+## jujubox and the local provider
 
- - Charmbox: The charmers dockerfile is for developing and reviewing charms:
-   *see
-   [charmbox instructions](charmbox.md)*.
+At this time you can not run the the local provider inside of Docker because 
+that is using nested containers. If you want to use the local provider install
+Juju on the host system.  
```

Comment:
This also likely needs a call out that its only available on trusty, with the juju-local package installed. This will never be true on windows or osx hosts